MachineLearning_Engine/MachineLearning_Engine.csproj

```diff
@@ -7,7 +7,7 @@
     <ProjectGuid>{61DCF1F4-D6DD-4512-8905-58D1868DA9DB}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>MachineLearning_Engine</RootNamespace>
+    <RootNamespace>BH.Engine.MachineLearning</RootNamespace>
     <AssemblyName>MachineLearning_Engine</AssemblyName>
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
I think all other BHoM toolkist are 4.5.2? I think this has come up before I know I have had issues when working with dlls from both.

Reply:
Yeah, that's correct, but unfortunately `4.6.2` is the minimum to be able to use external python packages. What problems was it creating? Do you remember?